Makefile

```diff
@@ -25,8 +25,9 @@ BUILD_OMS_VAL=0
 endif
 
 all:
-	rm -rf release/*.{rpm,deb};
+	rm -rf release/*.rpm release/*.deb
```

Comment:
`rm` in the `all` target is strange. Is there a good reason to build regardless of whether or not there is a change in the dependencies?